File: C Modules/Assignments/Assignment no 6/Question-1/Type_4/Assignment no 2/question-6.c
#include<stdio.h>
int divisible(int num)
{
    if(num%3==0 && num%5==0)
    {
            return 0;
        }
        else if (num%3==0)
        {
            return 1;
        }   
    
    else if (num%5==0)
        {
            return 2;
        }
        else{
            return 3;
        }
    }
int main()
{
    int num;
    printf("enter the number:");
    scanf("%d",&num);
    int result=divisible(num);
    if(result==0)
    {
            printf("Divisible by both");
    }
        else if(result==1){
             printf("Divisible by 3 but not by 5");
        }   
    
    else if (result==2)
        {
            printf("Divisible by 5 but not by 3");
            /* code */
        }
        else{
            printf("Divisible by None");
        }  
    
    return 0;
}

File: Java Module/Assignments in c to java/Assignment-02/question_6.java
public class question_6 {
public static void main(String[] args) {
    
    int num=3;
    // printf("enter the number:");
    // scanf("%d",&num);

    if (num==0) {
       System.out.printf("Enter a valid number!!");
    }
    else if(num%3==0)
    {
        if (num%5==0){
            System.out.printf("Divisible by both");
        }
        else{
            System.out.printf("Divisible by 3 but not by 5");
        }   
    }
    else{
        if (num%5==0)
        {
            System.out.printf("Divisible by 5 but not by 3");
            /* code */
        }
        else{
            System.out.printf("Divisible by None");
        }  
    }
}
}

